src/luxcore/luxparser/luxparse.y

```diff
@@ -17,7 +17,7 @@
  *   along with this program.  If not, see <http://www.gnu.org/licenses/>. *
  ***************************************************************************/
 
-%name-prefix "luxcore_parserlxs_yy"
+%name-prefix = "luxcore_parserlxs_yy"
```

Comment:
Can you check if replacing:

`%name-prefix "luxcore_parserlxs_yy"`

with:

`%define api.prefix {luxcore_parserlxs_yy}`

work for you ? It is apparently the right and latest way to declare the prefix.

Reply:
suggestion 1 works 

in luxparse.y i get error
/Users/drquader/Documents/GitHub/LuxCore/src/luxcore/luxparser/luxparse.y:20.9-18: syntax error, unexpected identifier, expecting string

Reply:
Hmm it's related to bison right ??

i found 

// Prefix all tokens
%define api.token.prefix {SCHEMA_}

Reply:
https://twitter.com/plusvic/status/960997335835824128

maybe use newer bison ? what version do you use ??

Reply:
what about a simple 
IF(APPLE) ? is this accessable there ??

%name-prefix = "luxcore_parserlxs_yy"

is correct for bison 2.3 that comes with OSX10.13

Reply:
using bison 3.4 from brew cellar works but says deprecated ..... but this is not a good solution ...

Reply:
Does not seem to be as trivial in bison :(

Reply:
Nevermind a

echo 'export PATH="/usr/local/opt/bison/bin:$PATH"' >> ~/.bash_profile

did the trick